reactrestaurant/src/components/badge.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Badge(props){
+    return(
+<span class="badge badge-secondary">12</span>
```

Comment:
this would a good opportunity to pass the prop for the badge count from its parent component.